src/main/java/com/laytonsmith/core/functions/EntityManagement.java

```diff
@@ -3639,4 +3640,91 @@ public CHVersion since() {
 			return CHVersion.V3_3_1;
 		}
 	}
+	
+	@api
+	public static class set_entity_glowing extends EntitySetterFunction {
+		public String getName() {
+			return "set_entity_glowing";
+		}
+
+		public String docs() {
+			return "void {Entity ID, boolean} If true, applies glowing effect to the entity";
+		}
+
+		public Construct exec(Target t, Environment environment, Construct... args) throws ConfigRuntimeException {
+			Static.getEntity(args[0], t).setGlowing(Static.getBoolean(args[1]));
+			return CVoid.VOID;
+		}
+
+		public Version since() {
+			return CHVersion.V3_3_2;
+		}
+	}
+
+	@api
+	public static class get_entity_glowing extends EntityGetterFunction {
+		public String getName() {
+			return "get_entity_glowing";
+		}
+
+		public String docs() {
+			return "boolean {Entity} Returns true if the entity is glowing";
+		}
+
+		public Construct exec(Target t, Environment environment, Construct... args) throws ConfigRuntimeException {
+			MCEntity e = Static.getEntity(args[0], t);
+			return CBoolean.GenerateCBoolean(e.isGlowing(), t);
+		}
+
+		public Version since() {
+			return CHVersion.V3_3_2;
+		}
+	}
+	
+	@api
+	public static class set_entity_gliding extends EntitySetterFunction {
+		public String getName() {
+			return "set_entity_gliding";
+		}
+
+		public String docs() {
+			return "void {Entity, boolean} If possible, makes the entity glide";
+		}
+
+		public Construct exec(Target t, Environment environment, Construct... args) throws ConfigRuntimeException {
+			MCLivingEntity e = Static.getLivingEntity(args[0], t);
+			boolean glide = Static.getBoolean(args[1]);
+
+			if (e.isOnGround()) {
+				throw new CREException("The specified entity cannot glide right now", t);
```

Comment:
If we're going to do this, it should probably be mentioned in the docs. Do we need to check if grounded? As far as I can tell, it would just revert to not gliding immediately.